src/Jackett.Common/Indexers/TorrentParadise.cs

```diff
@@ -0,0 +1,167 @@
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   cacheService: cs,
+                   configData: new ConfigurationData())
+        {
+            Encoding = Encoding.GetEncoding("UTF-8");
+            Language = "en-us";
+            Type = "public";
+
+            AddCategoryMapping(8000, TorznabCatType.Other);
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var query = new TorznabQuery { SearchTerm = DateTime.Now.Year.ToString(), IsTest = true };
+            var releases = await PerformQuery(query);
+
+            await ConfigureIfOK(string.Empty, releases.Any(),
+                                () => throw new Exception("Could not find releases from this URL"));
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var searchUrl = CreateSearchUrl(query);
+            if (string.IsNullOrWhiteSpace(searchUrl)) return new List<ReleaseInfo>();
+
+            var response = await RequestWithCookiesAndRetryAsync(searchUrl);
+
+            try
+            {
+                var results = JsonConvert.DeserializeObject<IEnumerable<TorrentParadiseResult>>(response.ContentString);
+                if (results == null) return new List<ReleaseInfo>();
+                return ConvertResultsIntoReleaseInfos(results);
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Error while parsing json: " + response.ContentString, ex);
+            }
+        }
+
+        private string CreateSearchUrl(TorznabQuery query)
+        {
+            var queryString = query.GetQueryString();
+            if (string.IsNullOrWhiteSpace(queryString))
+            {
+                if (!query.IsTest)
+                {
+                    return null;
+                }
+                query.SearchTerm = DateTime.Now.Year.ToString();
+                queryString = query.GetQueryString();
+            }
+
+            return ApiUrl + "search?q=" + queryString;
+        }
+
+        private IEnumerable<ReleaseInfo> ConvertResultsIntoReleaseInfos(IEnumerable<TorrentParadiseResult> results)
+        {
+            foreach (var result in results)
+            {
+                yield return ConvertResultIntoReleaseInfo(result);
+            }
+        }
+
+        private ReleaseInfo ConvertResultIntoReleaseInfo(TorrentParadiseResult result)
+        {
+            return new ReleaseInfo
+            {
+                Title = result.Text,
+                Size = result.Size,
+                Seeders = result.Seeders,
+                Peers = result.Seeders + result.Leechers,
+                MagnetUri = CreateMagnetUri(result),
```

Comment:
Remove the method `CreateMagnetUri`
If you set InfoHash = result.Id the magnet will be auto-generated.